Documentation/ApiOverview/Http/Index.rst

```diff
@@ -128,93 +141,87 @@ TYPO3 provides a small set of helper methods related to HTTP Requests in the cla
 HttpUtility::redirect
 ---------------------
 
-.. deprecated:: 11.3
-   With TYPO3 v11.3 this method has been deprecated. Create a direct response
-   using the :ref:`ResponseFactory <request-handling>` instead.
+..  deprecated:: 11.3
+    This method is deprecated and will be removed with TYPO3 v12.0. Create a
+    direct response using the :ref:`ResponseFactory <request-handling-psr-17>`
+    instead.
 
 .. _http_utility_response_migration:
 
 Migration
 ~~~~~~~~~
 
-Most of the time, a proper PSR-7 Response
-can be passed back to the call stack (request handler). Unfortunately there
-might still be some places, inside the call stack, where it is not possible to
-directly return a PSR-7 response. In such case, a :php:`PropagateResponseException`
-could be thrown. It will automatically be caught by a PSR-15 middleware and the
-given PSR-7 Response will then directly be returned.
+Most of the time, a proper PSR-7 response can be returned to the call stack
+(request handler). Unfortunately there might still be some places within the
+call stack where it is not possible to return a PSR-7 response directly. In such
+a case a :php:`TYPO3\CMS\Core\Http\PropagateResponseException` could be thrown.
+This is automatically caught by a PSR-15 middleware and the given PSR-7 response
+is then returned directly.
 
-.. code-block:: php
+..  code-block:: php
```

Comment:
Please add a caption. Where can this code be used?

Reply:
Maybe a middleware or a command. Middleware may not be the best example. Thought also about it, but it adds more boilerplate for both of them for this simple migration, so I wanted to avoid this.

Reply:
As this is also removed in v12.0 I would remove the example (including Migration) in main.

Reply:
agreed